ai2thor/controller.py

```diff
@@ -410,6 +411,22 @@ def __init__(
         self.add_depth_noise = add_depth_noise
         self.include_private_scenes = include_private_scenes
         self.x_display = None
+        self.gpu_device = gpu_device
+        cuda_visible_devices = list(
+            map(int, filter(lambda y: y.isdigit(),
+            map(lambda x: x.strip(),
+                os.environ.get('CUDA_VISIBLE_DEVICES', '').split(',')))))
+
+        if self.gpu_device:
```

Comment:
Won't this if statement be skipped if `gpu_device == 0`?

Reply:
Fixed.